scripts/ci/build_emscripten.sh

```diff
@@ -40,7 +40,7 @@ else
 	BUILD_DIR="$1"
 fi
 
-apt-get update && apt-get install lz4
+apt-get update && apt-get install lz4 --no-install-recommends
```

Comment:
We got `set -e` so no need for `&&`:
```suggestion
apt-get update
apt-get install lz4 --no-install-recommends
```